src/routes/+page.svelte

```diff
@@ -23,7 +23,8 @@
 				key: 'ABCDEFGH'[idx],
 				dynamic,
 				el: undefined,
-				value: 3
+				value: 3,
+				bgImage: `src/lib/assets/hero-${idx}.png`
```

Comment:
to accommodate multiple background images with various alignments, maybe here we can add a background class (eg `.hero-${idx}` or `.hero-${dynamic.short}`) and then in the style we can have bespoke styling instead of treating all of them exactly the same. 

In the [results page](https://github.com/acc-awcs/8-Dynamics/blob/584e602612ed742379f9c06af4febf228364e9d2/src/routes/results/%5Bresults_string%5D/%2Bpage.svelte#L28) I added multiple backgrounds with various alignment, in case that's helpful to see!

Reply:
Agreed! I think since we're dealing with a variety of different shapes it makes sense to style specific locations for each. I think especially moving images that intersect with the slider on mobile will be helpful (e.g. shifting the butterfly in the screenshot so it's below the slider). We can use media queries if we want to keep the desktop version closely matching designs.

<img width="438" alt="Screenshot 2024-12-06 at 2 00 05 PM" src="https://github.com/user-attachments/assets/a4c5fc52-bae7-4753-91f2-ca3db4f51fb7">